packages/@lwc/engine/src/framework/base-lightning-element.ts

```diff
@@ -257,7 +250,6 @@ function BaseLightningElementConstructor(this: LightningElement) {
         throw new ReferenceError();
```

Comment:
probably this be a better error, not a reference error, something like: `Invalid Construction` or somehting like that.